src/main/java/seedu/address/model/task/Status.java

```diff
@@ -30,6 +30,10 @@ public Status updateStatus() {
         return new Status(!isCompleted);
     }
 
+    public boolean isCompleted(){
```

Comment:
Very minor nickpick, remember to be consistent with a space `() {`
To make your life easier, you can download the Prettier extension on github, so that when you save, everything is formatted properly for you!

Reply:
Sure, will update my settings and keep this in mind for the future!